app/domain/logs.rb

```diff
@@ -360,6 +360,50 @@ module AuthnJwt
         code: "CONJ00070D"
       )
 
+      FetchingJwksUriConfigurationValue = ::Util::TrackableLogMessageClass.new(
```

Comment:
I do not found this `FetchingJwksUriConfigurationValue` in the code can you go over all the logs again and validate where do you print them ?

A general comment i think we should not have `jwks-uri` or `provider-uri` hardcoded in the messages because it may changes in the future:

For example WDYT to change to following: 
```
Fetching jwks-uri authenticator configuration value
Fetching provider-uri authenticator configuration value
```
Into
Fetching jwt signing key uri from '{0-resource-id}'

Reply:
I fixed the problem with the unused logs.

about the general comment: should I use the same logs for jwks-uri and provider-uri? 
and your example for the log message is really general, in that case that log will be very high level